src/Resource/Repository/RepositoryManager.js

```diff
@@ -0,0 +1,33 @@
+import createRepository from './RepositoryFactory';
+import Repository from './Repository';
+
+const repositories = new Map();
+
+const RepositoryManager = {};
+
+Object.assign(RepositoryManager, {
+  /**
+   * @param {HyralConnector} connector
+   * @param {String} resourceType
+   * @param {String} identifier='id'
+   *
+   * @returns HyralRepository
+   */
+  createRepository(connector, resourceType, identifier = 'id') {
+    return createRepository(RepositoryManager, Repository, connector, resourceType, identifier);
+  },
+  /**
+   * @param {HyralRepository} resourceRepository
+   */
+  addRepository(resourceRepository) {
+    repositories.set(resourceRepository.resourceType, resourceRepository);
+  },
+  /**
+   * @param resourceType
+   */
+  getRepository(resourceType) {
+    return repositories.get(resourceType);
+  },
+});
```

Comment:
Ik zou ook graag een getRepositories hebben, zodat in bijvoorbeeld de vuex store er 1 store module per repository gemaakt kan worden.